PovertyAction/research-resources
# Research Resources dev site update

# Pull Request Summary 🚀

## What does this PR do? 📝



Re-organizes existing and planned content for the Research Resources website. Also, the design of the website has been reformatted to better align with the look and feel of the Quarto Website (https://quarto.org/).

## Why is this change needed? 🤔



We want to set up files to be included in the Research Resources such that the structure of the site makes it easy to develop new content and makes the site navigation as clear as possible.

## How was this implemented? 🛠️



I moved website pages to the `docs/` sub-directory and updated the `_quarto.yml` file to better align with the look and navigation of the Quarto Website (https://quarto.org/).

## How to test or reproduce ? 🧪



I ran `just preview-docs` (or `quarto preview`) to review that the website builds as expected. I also ran `just build-docs` to render any of the quarto outputs.

Note that to reproduce, you need to have the relevant software installed, which you can do by running `just get-started`. That will install Python, R, and Quarto, which are needed to render the site content. You will also need Stata 18, if you want to render the `stata.qmd` page.

## Screenshots (if applicable) 📷



The following screenshots give a sense of what the new site design looks like:

<img width="1883" alt="image" src="https://github.com/user-attachments/assets/fc0fb3af-aea6-4ea5-b94c-716dd1339de2">

<img width="1885" alt="image" src="https://github.com/user-attachments/assets/125cbdba-c731-4776-a09b-02dd28ef5244">

<img width="1890" alt="image" src="https://github.com/user-attachments/assets/6c419014-a3e3-4ea2-814a-c0c08328ef3e">

## Checklist ✅

- [ ] I have limit this PR to less than 1000 lines of code change (if not, explain why)
> The large number of line changes are due to migrating files from the root directory to the `/docs` sub-directory. The substantive changes to the code are in the `_quarto.yml` file.
- [ ] I have updated/added tests to cover my changes (if applicable)
> Not applicable
- [X] I have updated/added requirements to cover my changes (if applicable)
- [X] I have reviewed and resolved any merge conflict

## Reviewer Emoji Legend

|     |         `:code:`          | Meaning                                                                                                                                                                                                                                         |
| :-: | :-----------------------: | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| 😃👍💯 | `:smiley:` `:+1:` `:100:` | I like this... <br /><br /> ...and I want the author to know it! This is a way to highlight positive parts of a code review.                                                                                                                    |
| ⭐⭐⭐ |  `:star: :star: :star:`   | Important to fix before PR can be approved... <br /><br /> And I am providing reasons why it needs to be addressed as well as suggested improvements.                                                                                           |
| ⭐⭐  |      `:star: :star:`      | Important to fix but non-blocking for PR approval... <br /><br /> And I am providing suggestions where it could be improved either in this PR or later.                                                                                         |
|  ⭐  |         `:star:`          | Give this some thought but non-blocking for PR approval... <br /><br /> ...and consider this a suggestion, not a requirement.                                                                                                                   |
|  ❓  |       `:question:`        | I have a question. <br /><br /> This should be a fully formed question with sufficient information and context that requires a response.                                                                                                        |
|  📝  |         `:memo:`          | This is an explanatory note, fun fact, or relevant commentary that does not require any action.                                                                                                                                                 |
|  ⛏  |         `:pick:`          | This is a nitpick. <br /><br /> This does not require any changes and is often better left unsaid. This may include stylistic, formatting, or organization suggestions and should likely be prevented/enforced by linting if they really matter |
| ♻️  |        `:recycle:`        | Suggestion for refactoring. <br /><br /> Should include enough context to be actionable and not be considered a nitpick.                                                                                                                        |
